packages/@aws-cdk/aws-servicecatalogappregistry/README.md

```diff
@@ -124,3 +124,57 @@ const myStack = new Stack(app, 'MyStack');
 declare const application: appreg.Application;
 application.associateStack(myStack);
 ```
+
+## Sharing
+
+You can share your AppRegistry applications and attribute groups with AWS Organizations, Organizational Units (OUs), AWS accounts within an organization, as well as IAM roles and users. AppRegistry requires that AWS Organizations is enabled in an account before deploying a share of an application or attribute group.
+
+### Sharing an application
+
+```ts
+import * as iam from '@aws-cdk/aws-iam';
+declare const application: appreg.Application;
+declare const myRole: iam.IRole;
+declare const myUser: iam.IUser;
+application.shareResource({
+  accounts: ['123456789012'],
+  organizations: ['arn:aws:organizations::123456789012:organization/o-my-org-id'],
+  roles: [myRole],
+  users: [myUser]
+});
+```
+
+E.g., sharing an application with multiple accounts:
+
+```ts
+import * as iam from '@aws-cdk/aws-iam';
+declare const application: appreg.Application;
+application.shareResource({
+  accounts: ['123456789012', '234567890123'],
+});
+```
+
+### Sharing an attribute group
```

Comment:
Since an attribute group is attached to an application, why is the group not automatically shared right away?

Reply:
I would also like to know the answer to this question. In addition, resources are also attached to applications; is it possible to share an individual resource?

Reply:
Currently, AppRegistry does not automatically share Attribute Groups or Resources that are associated with an application when a share is created for that application. To bring parity between the sharing experience in the console and the experience in CDK, in my opinion we don't need to implement automatic sharing of attribute groups or resources in this PR. Since CDK is a great abstraction layer to offer such a feature, it could be implemented in an additional PR after more careful consideration.

Reply:
I'm fine with punting this to another PR, but this would be a very nice feature for CDK